src/core/isSame.js

```diff
@@ -10,4 +12,4 @@ function isSame(x, y) {
   return x !== x && y !== y
 }
 
-module.exports = isSame
+module.exports = curry(isSame)
```

Comment:
@bennypowers - We should probably do this in `src/predicates` rather than here for consistency and perf reasons.

Reply:
np but in that case should we do the same with `isSameType`?